gems/pending/Gemfile

```diff
@@ -31,18 +33,17 @@ gem "memory_buffer",           ">=0.1.0",           :require => false
 gem "more_core_extensions",    "~>1.2.0",           :require => false
 gem "net-sftp",                "~>2.1.2",           :require => false
 gem "nokogiri",                "~>1.6.0",           :require => false
-gem "ovirt",                   "~>0.6.0",           :require => false
-gem "kubeclient",              "=0.1.17",           :require => false
 gem "openshift_client",        "=0.0.8",            :require => false
-gem "rest-client",                                  :require => false, :git => "git://github.com/rest-client/rest-client.git", :ref => "08480eb86aef1e"
+gem "ovirt",                   "~>0.6.0",           :require => false
 gem "parallel",                "~>0.5.21",          :require => false
 gem "Platform",                "=0.4.0",            :require => false
+gem "psych",                   "~>2.0.12"
+gem "rest-client",                                  :require => false, :git => "git://github.com/rest-client/rest-client.git", :ref => "08480eb86aef1e"
 gem "rubyzip",                 "=0.9.5",            :require => false  # TODO: Review 0.9.7 breaking log collection in FB14646
+gem "rubyipmi",                "~>0.10.0",          :require => false
 gem "rufus-lru",               "~>1.0.3",           :require => false
-gem "uuidtools",               "~>2.1.3",           :require => false
 gem "trollop",                 "~>2.0",             :require => false
-gem "psych",                   "~>2.0.12"
-gem "apipie-bindings",         "~>0.0.12",          :require => false
+gem "uuidtools",               "~>2.1.3",           :require => false
```

Comment:
@brandondunne it's hard to tell what changed here without opening each of the commits.  It looks like we dropped the rubyipmi gem?  Maybe we should leave the sorting to a separate PR in the future as it makes it hard to review?  Thoughts?